core/platform/cloud_logging/cloud_logging_service.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides logging functionality from Google Cloud Logging."""
+
+import google.cloud.logging
+
+
+def enable_cloud_logging():
+    """Enables google cloud logging."""
+
+    # Instantiates a client
+    client = google.cloud.logging.Client()
+
+    # Retrieves a Cloud Logging handler based on the environment
+    # you're running in and integrates the handler with the
+    # Python logging module. By default this captures all logs
+    # at INFO level and higher
```

Comment:
Are we fine with this? Do we only use INFO and higher? Or do we in some cases use debug logging?

Reply:
I have changed the level to debug.